Lib/test/test_ssl.py

```diff
@@ -155,7 +155,8 @@ def seclevel_workaround(*ctxs):
         for ctx in ctxs:
             if (
                 hasattr(ctx, "minimum_version") and
-                ctx.minimum_version <= ssl.TLSVersion.TLSv1_1
+                ctx.minimum_version <= ssl.TLSVersion.TLSv1_1 and
+                ctx.security_level > 1
```

Comment:
Is `security_level` defined on all context objects? (I don't know why we guard the `minimum_version` here)

Reply:
I believe `security_level` is [always](https://github.com/python/cpython/blob/2e96f5ae4d1700d77a53180b4866c5b9f002e62d/Modules/_ssl.c#L3786C21-L3786C36) defined, and [it is read-only](https://docs.python.org/3/library/ssl.html#ssl.SSLContext.security_level). `minimum_version`, on the other hand [is not](https://docs.python.org/3/library/ssl.html#ssl.SSLContext.minimum_version) and as such can be set to `None` by the caller to fall back to [`TLSVersion.MINIMUM_SUPPORTED`](https://docs.python.org/3/library/ssl.html#ssl.TLSVersion.MINIMUM_SUPPORTED).